src/geoaxis.jl

```diff
@@ -829,6 +829,16 @@ function Makie.plot!(axis::GeoAxis, plot::Makie.AbstractPlot)
     return plot
 end
 
+function Makie.MakieCore._create_plot!(F, attributes::Dict, ax::GeoAxis, args...)
+    source = pop!(attributes, :source, nothing)
+    dest = pop!(attributes, :dest, nothing)
+    plot = Plot{Makie.default_plot_func(F, args)}(args, attributes)
+    isnothing(source) || (plot.kw[:source] = source)
+    isnothing(dest) || (plot.kw[:dest] = dest)
+    Makie.plot!(ax, plot)
+    return plot
+end
+
```

Comment:
Perfect, thanks!!  I wasn't sure exactly where to inject this but this looks great.